src/entities/ensRegistration.ts

```diff
@@ -3,21 +3,37 @@ import { ENS_RECORDS } from '@rainbow-me/helpers/ens';
 
 export type Records = { [key in keyof typeof ENS_RECORDS]: string };
 
-export interface RegistrationParameters {
-  name: string;
+export interface ENSRegistrationRecords {
+  coinAddress: { key: string; address: string }[] | null;
+  contentHash: string | null;
+  ensAssociatedAddress: string | null;
+  text: { key: string; value: string }[] | null;
+}
+
+export interface TransactionRegistrationParameters {
+  commitTransactionHash?: string;
+  commitTransactionConfirmedAt?: number;
+  registerTransactionHash?: number;
+}
+
+export interface RegistrationParameters
+  extends TransactionRegistrationParameters {
   duration: number;
-  records: Records;
+  name: string;
   ownerAddress: EthereumAddress;
   rentPrice: string;
-  salt?: string;
+  records?: ENSRegistrationRecords;
```

Comment:
thinking about it... maybe this format will be nice actually, it will make dealing with the edit form a lot easier too... we can open another PR to do this maybe?